plansys2_domain_expert/src/plansys2_domain_expert/DomainExpertNode.cpp

```diff
@@ -117,7 +117,7 @@ DomainExpertNode::on_configure(const rclcpp_lifecycle::State & state)
   domain_expert_ = std::make_shared<DomainExpert>(domain_str);
 
   bool check_valid = planner->is_valid_domain(domain_expert_->getDomain(), get_namespace());
-  if (!check_valid) {
+  if (check_valid) {
```

Comment:
I think that the correct semantic should be `if not valid then print error`

Reply:
Opps. I reverted this change. ... For some reason I still need to do this on my Foxy build, but I assume it is something wrong on my end. ... Previously I was able to resolve this by switching to the ros2 branch of popf. However, that does not seem to work any longer. I will try upgrading to Humble soon.